examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -14498,7 +14498,7 @@
               "code": 4,
               "mfgCode": null,
               "side": "server",
-              "type": "OperationalStateStruct",
+              "type": "OperationalStateEnum",
```

Comment:
why did this not trigger any .matter file changes?

Reply:
Sure, fixes this bug.